src/main/Main.java

```diff
@@ -3,10 +3,14 @@
 import main.Database.Database;
 import main.Panels.UserPanel;
 
+import javax.xml.crypto.Data;
+
 public class Main
 {
     public static void main(String[] args)
     {
-        Application.appMain();
+        Database db = new Database();
```

Comment:
Let the application hold a detabase instance. Instaed of Passing it as an argument.